src/android/org/sunbird/support/SunbirdFileHandler.java

```diff
@@ -59,11 +68,134 @@ public static String makeEntryInSunbirdSupportFile(String packageName, String ve
         return filePath;
     }
 
+    public static String makeEntryInSunbirdConfiguration(String packageName, String versionName, String appName,
+                                                         Context context) throws IOException {
+        File supportDirectory = SunbirdFileHandler.getRequiredDirectory(Environment.getExternalStorageDirectory(),
+                appName + SUPPORT_DIRECTORY);
+        String filePath = supportDirectory + "/" + appName + CONFIG_FILE;
+        SunbirdFileHandler fileHandler = new SunbirdFileHandler();
+
+        // for the first time when file does not exists
+        if (!SunbirdFileHandler.checkIfFileExists(filePath)) {
+            SunbirdFileHandler.createFileInTheDirectory(filePath);
+            String versionHistory = versionName + SEPERATOR + System.currentTimeMillis() + SEPERATOR + "1";
+            String firstEntry = fileHandler.getSunbirdConfigurations(context) + "sv: " + versionHistory;
+            SunbirdFileHandler.saveToFile(filePath, firstEntry);
+        } else {
+            String lastLineOfFile = SunbirdFileHandler.readLastLineFromFile(filePath);
```

Comment:
You don't really have to do this as like above method makeEntryInSunbirdSupportFile, all you need to do is call getSunbirdConfigurations() and it should get you all the required details.